sdk/digitaltwins/Azure.DigitalTwins.Core/src/RequestOptions.cs

```diff
@@ -13,13 +9,18 @@ namespace Azure.DigitalTwins.Core
     public class RequestOptions
     {
         /// <summary>
-        /// A string representing a weak ETag for the entity that this request performs an operation against, as per RFC7232. The request's operation is performed
-        /// only if this ETag matches the value maintained by the server, indicating that the entity has not been modified since it was last retrieved.
-        /// To force the operation to execute only if the entity exists, set the ETag to the wildcard character '*'. To force the operation to execute unconditionally, leave this value null.
+        /// A string representing a weak ETag for the entity that this request performs an operation against, as per RFC7232.
         /// </summary>
         /// <remarks>
-        /// If this value is not set, it defaults to null, and the ifMatch header will not be sent with the request. This means that update and delete will be unconditional and the operation will execute regardless of the existence of the resource.
+        /// <para>
+        /// The request's operation is performed only if this ETag matches the value maintained by the server,
+        /// indicating that the entity has not been modified since it was last retrieved.
+        /// To force the operation to execute only if the entity exists, set the ETag to the wildcard character '*'.
+        /// To force the operation to execute unconditionally, leave this value null.
+        /// If this value is not set, it defaults to null, and the ifMatch header will not be sent with the request.
+        /// This means that update and delete will be unconditional and the operation will execute regardless of the existence of the resource.
+        /// </para>
         /// </remarks>
-        public string IfMatchEtag { get; set; } = null;
+        public string IfMatch { get; set; } = null;
```

Comment:
I know this was already set this way, but string already defaults to null, so no reason to initialize.